packages/windows/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.27.0"
+  changes:
+    - description: Adding  initial Windows AppLocker data stream [beta]
```

```suggestion
    - description: Adding initial Windows AppLocker data stream [beta]
```
I'm not sure the `[beta]` is helpful here, but defer to others.